src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_LocalFunctions.cs

```diff
@@ -45,7 +45,7 @@ public AbstractLocalFunctionState(TLocalState stateFromBottom, TLocalState state
 
         protected abstract TLocalFunctionState CreateLocalFunctionState();
 
-        private SmallDictionary<LocalFunctionSymbol, TLocalFunctionState>? _localFuncVarUsages = null;
+        protected SmallDictionary<LocalFunctionSymbol, TLocalFunctionState>? _localFuncVarUsages = null;
```

Comment:
This is a delay initialized value. Exposing it directly via `protected` means that derived classes need to be aware of this implementation detail. That seems fragile. If the intent is for derived types to have raw usage and effectively ignore the delay init I think it's better to expose that via a named method like `GetLocalFuncUsageTable` that has a pair of `GetOrCreate`.